kuberos.go

```diff
@@ -21,13 +21,14 @@ import (
 const (
 	// DefaultKubeCfgEndpoint is the default endpoint to which clients should
 	// be redirected after authentication.
-	DefaultKubeCfgEndpoint = "/ui"
+	DefaultKubeCfgEndpoint = "ui"
 
 	schemeHTTP  = "http"
 	schemeHTTPS = "https"
 
 	elbHeaderForwardedProto = "X-Forwarded-Proto"
 	elbHeaderForwardedFor   = "X-Forwarded-For"
+	headerForwardedPrefix   = "X-Forwarded-Prefix"
```

Comment:
A nitpick: would you mind renaming `elbHeaderForwarded` to `headerELBForwarded` in the above two lines. That way all the header related constants will start with `header`.

Reply:
I think it would be better to remove the `elb` part all together, since those headers are not AWS specific. So I converted that to `headerForwarded`